Content.Client/Players/PlayTimeTracking/JobRequirementsManager.cs

```diff
@@ -90,33 +92,64 @@ private void RxJobWhitelist(MsgJobWhitelist message)
         Updated?.Invoke();
     }
 
+    /// <summary>
+    /// Check the job prototype against the current player, for requirements and bans
+    /// </summary>
     public bool IsAllowed(JobPrototype job, HumanoidCharacterProfile? profile, [NotNullWhen(false)] out FormattedMessage? reason)
+    {
+        var list = new List<string>{JobPrefix + job.ID};
+        return IsAllowed(list, profile, out reason);
+    }
+
+    /// <summary>
+    /// Check each job/antag prototype against the current player, for requirements and bans
+    /// The prototypes MUST start with the job/antag prefix
+    /// </summary>
+    public bool IsAllowed(List<string> prototypes, HumanoidCharacterProfile? profile, [NotNullWhen(false)] out FormattedMessage? reason)
```

Comment:
It seems unnecessary for this to just be strings, what about something like

```cs
enum RoleType {
  Job,
  Antag,
}
```

and then you pass a a List of tuples with the job type and the name. 
It could also just be a boolean (`isAntag`) if it's unambiguous enough maybe.

Reply:
job:<jobProtoId> is the format already used to record and store all existing (job) rolebans, seems reasonable to use it